internal/daysteps/daysteps.go

```diff
@@ -12,9 +19,52 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	strings := strings.Split(data, ",")
+	if len(strings) != 2 {
+		log.Println(errors.New("parsePackage: len(strings) !=2"))
+		return 0, 0, errors.New("parsePackage: len(strings) !=2")
+	}
+	steps, err := strconv.Atoi(strings[0])
+	if err != nil {
+		log.Println(err)
+		return 0, 0, errors.New("parsePackage: cant convert steps to string")
+	}
+	if steps < 1 {
+		log.Println(err)
+		return 0, 0, errors.New("parsePackage: steps <= 0")
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```